### DISCUSSION TITLE
getCell().value returning incorrect result

### DISCUSSION THREAD

**Discussion by unitedvetcaretech** [exceljs/exceljs#1965] [Q&A]
Hello all --

I have a sheet with a cell that has a simple sum() formula in it. I've verified this sheet was created and saved in Excel.
When I read this cell from the sheet, I get a result of `{ formula: 'SUM(F7:F49)', result: 123456.688888 }`, which is much higher than what it displays at in excel, and much higher than it should be. Any thoughts on how I can start tracking this down?

Thanks!

**Comment by Tomas2D** [comment]
Probably you are passing a string to the cell value and not a number.
